kinto_http/__init__.py

```diff
@@ -259,6 +267,89 @@ def delete_buckets(self, safe=True, if_match=None, if_exists=False):
         resp, _ = self.session.request('delete', endpoint, headers=headers)
         return resp['data']
 
+    # Groups
+
+    def get_groups(self, bucket=None):
+        endpoint = self.get_endpoint('groups', bucket=bucket)
+        return self._paginated(endpoint)
+
+    def create_group(self, group=None, bucket=None,
```

Comment:
Doing a POST is giving the error message : "Method not allowed for this endpoint". Am I doing something wrong?